scst-store/deployment-details.hbs.md

```diff
@@ -20,6 +20,40 @@ The installation creates the following in your Kubernetes cluster:
 
 ## <a id='configuration'></a> Deployment configuration
 
+### Supported Network Configurations
+
+VMware recommended connection methods based on Tanzu Application Platform setup:
+
+* Single or multi-cluster with Contour = `Ingress`
+* Single cluster without Contour and with `LoadBalancer` support = `LoadBalancer`
+* Single cluster without Contour and without `LoadBalancer` = Port forwarding
+* Single cluster without Contour, without `LoadBalancer` and user does not have port forwarding access = `NodePort`
+* Multi-cluster without Contour = Not supported
+
+For a production environment, VMware recommends that Supply Chain Security Tools - Store is installed with ingress enabled.
+
+#### <a id='appserv-type'></a>App service type
+
+Supported values include `LoadBalancer`, `ClusterIP`, `NodePort`. The `app_service_type` is set to `LoadBalancer` by default. If your environment does not support `LoadBalancer`, and you want to use `ClusterIP`, configure the `app_service_type` property in your `metadata-store-values.yaml`:
+
+```yaml
+app_service_type: "ClusterIP"
+```
+
+If the `ingress_enabled` property is set to `"true"`, VMware recommends setting the `app_service_type` property to `"ClusterIP"`.
+
+#### <a id='ingress'></a>Ingress support
+
+Supply Chain Security Tools - Store's values file allows you to enable ingress support and to configure a custom domain name to use Contour to provide external access to Supply Chain Security Tools - Store's API. For example:
+
+```yaml
+ingress_enabled: "true"
+ingress_domain: "example.com"
+app_service_type: "ClusterIP" # recommended setting
+```
+
+An HTTPProxy object is then installed with `metadata-store.example.com` as the fully qualified domain name. See [Ingress and multicluster support](ingress-multicluster.md).
```

Comment:
In the latest upstream, I have separated the two docs. Now there is `ingress.hbs.md` and `multicluster-setup.hbs.md`.